state/state.go

```diff
@@ -99,7 +99,22 @@ func (s *State) Message() string {
 	return "paused"
 }
 
-// formatDuration format a duration to 12:34
+// StatusMessage returns a message describing the state of a session.
+func (s *State) StatusMessage() string {
+	msgs := map[byte]string{
+		'w': "working session",
+		's': "short break",
+		'l': "long break",
+	}
+
+	if !s.IsEnded() {
+		return fmt.Sprintf("A %s is in progress", msgs[s.pattern[s.currentIdx]])
+	}
+
+	return fmt.Sprintf("Your %s has ended, time for a %s.", msgs[s.pattern[s.currentIdx]], msgs[s.pattern[s.currentIdx+1]])
```

Comment:
We need to get `msgs[s.pattern[0]]` for the next message if `s.currentIdx == len(s.pattern)-1`.

Reply:
Thank you for the feedback and sorry for the delay in fixing it. :bow: